Converter/include/PotreeConverter.h

```diff
@@ -77,14 +81,14 @@ inline ScaleOffset computeScaleOffset(Vector3 min, Vector3 max, Vector3 targetSc
 
 
 

```

Comment:
Again, an explanation of the inteneded functionality would help subsequent users/devs. This appears to be an attribute parser for variable length records iin las/laz files but some documentation would help

Reply:
Same comment as above